rollup/runtime/Cargo.toml

```diff
@@ -21,6 +21,7 @@ hex-literal = { version = "0.3.1" }
 log = { version = "0.4.17", default-features = false }
 smallvec = "1.6.1"
 static_assertions = "1.1.0"
+alloy-sol-types = { version = "0.5.0", default-features = false }
```

Comment:
i think thats what causes problem, please include alloy-sol-types in `std` feature somewhere lower in that file